tsconfig.base.json

```diff
@@ -18,7 +18,7 @@
       "@griffel/babel-preset": ["packages/babel-preset/src/index.ts"],
       "@griffel/core": ["packages/core/src/index.ts"],
       "@griffel/devtools": ["packages/devtools/src/index.ts"],
-      "@griffel/e2e-typescript": ["e2e/typescript/src/index.ts"],
```

Comment:
These is no sense to have `@griffel/e2e-typescript` there as it exports nothing.